### PR TITLE
[FSDE-1419] Add help icon support in App layout WC

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/FSDE-1419

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-131/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://ff9220c3110fb059a0254c0d4d6658babc285a9b-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-131/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://ff9220c3110fb059a0254c0d4d6658babc285a9b-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-131/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://fec8d5a8c90f099519b504b8afc2a424e2a53c50-chi-pr.enterprise.ctl.io

**Review by mattnickles — APPROVED** [review]

File: src/website/views/templates/app-layout.pug [hunk 1]
```diff
@@ -279,17 +279,898 @@ wide_container: true
...
+          a(
+            href=`#html-help-icon`
+            role="tab"
+            aria-selected="false"
+            tabindex="-1"
+            aria-controls="html-help-icon"
+          ) HTML Blueprint
+    .chi-tabs-panel.-active#webcomponent-help-icon
+      .chi-tab__description
+        | Use the <code>title=""</code> attribute to display a title above the application layout.
```
Comment (janibolkvadze) [inline src/website/views/templates/app-layout.pug:374]: Do we need to repeat this?

File: src/website/views/templates/app-layout.pug [hunk 2]
```diff
@@ -279,17 +279,898 @@ wide_container: true
...
+
+    .chi-tabs-panel#html-help-icon
+      :code(lang="html")
+        <div class="chi-main">
+          <div class="chi-main__header">
+            <div class="chi-main__header-start">
+              <div class="chi-main__title">
+                <div class="-text--h3 -text--boldest -text--navy -m--0 -pr--2">
+                  App title
+                  <chi-button id="example__help-button" type="icon" size="sm" variant="flat" alternative-text="Help" slot="help-icon">
```
Comment (janibolkvadze) [inline src/website/views/templates/app-layout.pug:483]: Do we need to use web components in HTML blueprint tab?

File: src/website/views/templates/app-layout.pug [hunk 3]
```diff
@@ -279,17 +279,898 @@ wide_container: true
...
+          a(
+            href=`#html-help-icon-lumen`
+            role="tab"
+            aria-selected="false"
+            tabindex="-1"
+            aria-controls="html-help-icon-lumen"
+          ) HTML Blueprint
+    .chi-tabs-panel.-active#webcomponent-help-icon-lumen
+      .chi-tab__description
+        | Use the <code>title=""</code> attribute to display a title above the application layout.
```
Comment (janibolkvadze) [inline src/website/views/templates/app-layout.pug:675]: Same here

File: src/website/views/templates/app-layout.pug [hunk 4]
```diff
@@ -279,17 +279,898 @@ wide_container: true
...
+
+    .chi-tabs-panel#html-help-icon-lumen
+      :code(lang="html")
+        <div class="chi-main">
+          <div class="chi-main__header">
+            <div class="chi-main__header-start">
+              <div class="chi-main__title">
+                <div class="-text--h3 -text--boldest -text--navy -m--0 -pr--2">
+                  App title
+                  <chi-button id="example__help-button" type="icon" size="sm" variant="flat" alternative-text="Help" slot="help-icon" class="-ml--1">
```
Comment (janibolkvadze) [inline src/website/views/templates/app-layout.pug:784]: Same here

File: src/website/views/templates/app-layout.pug [hunk 5]
```diff
@@ -1586,15 +2471,30 @@ wide_container: true
...
+    .chi-tabs-panel#html-centurylink-help-icon
       :code(lang="html")
         <div class="chi-main">
           <div class="chi-main__header">
             <div class="chi-main__header-start">
               <div class="chi-main__title">
-                <div class="-text--h3 -text--boldest -m--0 -pr--2">App title</div>
+                <div class="-text--h3 -text--boldest -m--0 -pr--2">
+                  App title
+                  <chi-button id="example__help-button" type="icon" size="sm" variant="flat" alternative-text="Help" slot="help-icon" class="-ml--1">
```
Comment (janibolkvadze) [inline src/website/views/templates/app-layout.pug:2491]: Same here

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://727d5aacd1f202b099dff69aa2daf3d0061b7a40-chi-pr.enterprise.ctl.io

**Review by janibolkvadze — APPROVED** [review]

**Pull request merged**